estate_account/__manifest__.py

```diff
@@ -0,0 +1,9 @@
+{
+        'name': 'estate_account',
+        'depends': ['base', 'estate', 'account'],
```

Comment:
`base`is already a dependency of both `estate` and `account` it is redundant to put it here again